health_check/contrib/celery/backends.py

```diff
@@ -23,5 +23,7 @@ def check_status(self):
                 self.add_error(ServiceReturnedUnexpectedResult("Celery returned wrong result"))
         except IOError as e:
             self.add_error(ServiceUnavailable("IOError"), e)
+        except NotImplementedError as e:
+            self.add_error(ServiceUnavailable("NotImplementedError"), e)
```

Comment:
Could you add a more descriptive message? I know the other exceptions aren't handled much better, but this is a good place to start ;)